docs/mkdocs.yml

```diff
@@ -2,6 +2,12 @@ site_name: FakeItEasy
 site_dir: artifacts/docs
 repo_url: https://github.com/FakeItEasy/FakeItEasy
 extra_css: [css/fakeiteasy.css]
+docs_dir: .
+exclude_docs: |
+    pyproject.toml
+    uv.lock
+    mkdocs.yml
+    .python-version

```

Comment:
I'm a freak, but what would happen if these filenames were sorted?

Reply:
The Universe's wave function would probably collapse. Worth trying!

Reply:
Done. We're still here, so the experiment is inconclusive.